app/build.gradle.kts

```diff
@@ -15,13 +15,13 @@
  */
 plugins {
     id("com.android.application")
-    id("de.timfreiheit.resourceplaceholders.plugin")
+    alias(libs.plugins.resourceplaceholders) apply true
```

Comment:
Why we do this on resourceplaceholders and not on the lines next to this one? I'm not an expert on GVC so just asking

Reply:
The need to do this for some plugins relates to using the [Gradle plugins DSL](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block), rather than being specific to Version Catalogs - GVC just encourages use of `plugins`.

Some plugins do not manage their classpath/class loading in quite the way that the newer plugins DSL expects. For such plugins it is sometimes necessary to first load the plugin in the root Gradle project (bring it into the classpath before the `app` module is loaded) and then continue to actually apply it in the `app` (or other) module as required.

Note that in the root project `plugins` block I added:
```
alias(libs.plugins.resourceplaceholders) apply false // Android Extension in `app` breaks without this
```